src/components/UI/Layout/Navigation/SideMenus/SideMenus.module.css

```
@@ -37,12 +37,14 @@
   font-size: 13px;
   font-weight: 700;
   color: #073f24;
+  font-family: 'Heebo', sans-serif !important;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Heebo" (font-family-name-quotes)

(font-family-name-quotes)